packages/dart_frog_test/README.md

```diff
@@ -0,0 +1,108 @@
+# Dart Frog Test
+
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+[![License: MIT][license_badge]][license_link]
+
+A testing library that makes it easy to test Dart Frog services. It offers helpers to mock requests as well as
+custom matchers in order to write readable expectations.
+
+> [!NOTE]
+> This package is still experimental and although it is ready to be used, some/or all of its API
+> might change (with deprecations) in future versions.
```

Comment:
Does the `[!NOTE]` render similarly in Pub as in GitHub?

Reply:
Not really sure, I know that github markdown is way advanced.

Reply:
Let's find out then :eyes: